src/pages/messaging/agent-chat-api/v3.5/index.mdx

```diff
@@ -1578,6 +1578,10 @@ https://api.livechatinc.com/v3.5/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given **thread**. 
+
+[Read how to list all available chats](https://my.livechatinc.com/settings/tags) and [create a new tag](/management/configuration-api/v2.0#add-a-tag). In this method, letter case needs to be respected for tag names.
```

Comment:
1. For now, 'create a new tag' is linked to the v3.2 as I believe the v3.5 has not been published yet. Please correct me if I am wrong.
2. Please let me know if this information about the letter case is enough.

Reply:
1. Could you please link to v3.5? We will first release the v3.5 PR, and then yours, so that the link works. I suggested a link change.
2. We could go with the sentence proposed by @gluszczyk "In this method, `tag` is case sensitive." It better because it mentions the `tag` field explicitly.